apps/console/src/features/email-templates/api/email-templates.ts

```diff
@@ -237,3 +237,27 @@ export const replaceLocaleTemplateContent = (
             return Promise.reject(error);
         });
 };
+
+/**
+ * Get branding preferences of the tenant.
+ */
+export const getBrandingPreferences = ()
```

Comment:
Move `features/branding/api/branding-preferences.ts`

And use useSWR instead of GET.

Reply:
Implement a hook `useBrandingPreference`.